src/Exception/TooManyRequestsHttpException.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Akeneo\Pim\ApiClient\Exception;
+
+/**
+ * @copyright 2022 Akeneo SAS (https://www.akeneo.com)
+ * @license   https://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class TooManyRequestsHttpException extends ClientErrorHttpException
+{
+    /**
+     * How much time must be waited before next request. Result in seconds.
+     */
+    public function getRetryAfter(): int
+    {
+        $retryAfterHeader = $this->getResponse()->getHeader('Retry-After');
+
+        return (int) $retryAfterHeader[0];
+    }
+
+    /**
+     * How much time must be waited before next request. Result in milliseconds.
+     */
+    public function getRetryIn(): int
```

Comment:
- It's coupled to our Serenity edition and would fail with most of the CE install as this header is not in a standard
-Also, wouldn't it better in term of DX to wait directly in the HTTP client instead of letting this responsibility to the user?
- I don't see the point to have 2 methods in seconds and ms. Could we just expose one method `retryAfterInSeconds` or `retryAfterInMilliseconds`? It would avoid to read the phpdoc comment, which is generally a code smell.

Reply:
- Good to know
- Planned to be done afterward (https://akeneo.atlassian.net/browse/API-1702)
- Yes, let's keep it to the standard `Retry-After` header

Reply:
2nd point: Had a bit of conversation with Aymeric and Romain, we are unsure about whether this would be relevant and if it is, the way we should implement it. So we put the subject aside from now.